<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inspect3DProfileWidget</class>
 <widget class="QWidget" name="Inspect3DProfileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inspect3DProfileWidget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView_dlp1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>350</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_profChartRow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Row行数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_row">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_col">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Col列数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inspect">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>530</y>
     <width>341</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>检测</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_dlp4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>350</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_dlp3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>350</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_dlp2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>350</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_profChartCol">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>570</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
